source/slang/slang-ir-byte-address-legalize.cpp

```diff
@@ -478,6 +478,26 @@ struct ByteAddressBufferLegalizationContext
         return m_builder.emitIntrinsicInst(type, op, elementVals.getCount(), elementVals.getBuffer());
     }
 
+    IRInst* emitVectorizedLoad(IRType* type, IRInst* buffer, IRInst* baseOffset, IRIntegerValue immediateOffset, IROp op, IRType* elementType, IRIntegerValue elementCount)
```

Comment:
This should be named `emitLoadWithKnownOffset`, because it only vectorizes things if the offset is aligned.